src/middlewared/middlewared/plugins/failover_/disabled_reasons.py

```diff
@@ -78,6 +80,14 @@ def get_local_reasons(self, app, ifaces, reasons):
         if self.middleware.call_sync('failover.config')['disabled']:
             reasons.add(DisabledReasonsEnum.NO_FAILOVER.name)
 
+        current_node = self.middleware.call_sync('failover.node')
+        reboot_info = self.middleware.call_sync('failover.reboot.info')
+        if current_node in ('A', 'B') and reboot_info.pop('reboot_required'):
+            if reboot_info.pop(f'node_{current_node.lower()}_reboot_required'):
+                reasons.add(DisabledReasonsEnum.LOC_FIPS_REBOOT_REQ.name)
+            if list(reboot_info.values())[0]:
```

Comment:
This will always show local and remote needs to be rebooted.

Let's be explicit about the checks please so it's easier to read for other developers
```
current_node = self.middleware.call_sync('failover.node')
rinfo = self.middleware.call_sync('failover.reboot.info')
if rinfo[f'node_{current_node.lower()}_reboot_required']:
    reasons.add(DisabledReasonsEnum.LOC_FIPS_REBOOT_REQ.name)

if any((
    rinfo['node_a_reboot_required'] and current_node != 'A',
    rinfo['node_b_reboot_required'] and current_node != 'B',
)):
    reasons.add(DisabledReasonsEnum.REM_FIPS_REBOOT_REQ.name)